internal/validate/string_test.go

```diff
@@ -0,0 +1,69 @@
+package validate
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/getkin/kin-openapi/openapi3"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	"gotest.tools/v3/assert"
+)
+
+func TestStringRule_Validate(t *testing.T) {
+	t.Run("min length", func(t *testing.T) {
+		r := &ExampleRequest{RequiredString: "a"}
+		err := Validate(r)
+		assert.ErrorContains(t, err, "length (1) must be at least 2")
+	})
+	t.Run("max length", func(t *testing.T) {
+		r := &ExampleRequest{RequiredString: "abcdefghijklm"}
+		err := Validate(r)
+		assert.ErrorContains(t, err, "length (13) must be no more than 10")
```

Comment:
Thoughts on this error message? Any suggestions for how we can improve it?

Reply:
`length of string (x) exceeds maximum length of y`

maybe

Reply:
Adding "of string" sounds great! 

"exceeds maximum length" vs "must be no more than" is interesting. I guess we want a consistent feel for all of these messages. I'll try to compile them all into one place and play around with some of these options.